test/migration/verify/verify_migration_test.go

```diff
@@ -261,6 +261,14 @@ func verifyAdditionalDeploymentsCreatedUsingSSA(t *testing.T, awaitilities *wait
 	t.Run("verify registration service deployed using SSA in host", func(t *testing.T) {
 		testDeployment(t, "registration-service", awaitilities.Host().RegistrationServiceNs, "host-operator", "kubesaw-host-operator")
 	})
+
+	t.Run("verify webhook deployed using SSA in member1", func(t *testing.T) {
+		testDeployment(t, "member-operator-webhook", awaitilities.Member1().Namespace, "member-operator", "kubesaw-member-operator")
+	})
+
+	t.Run("verify webhook deployed using SSA in member2", func(t *testing.T) {
+		testDeployment(t, "member-operator-webhook", awaitilities.Member2().Namespace, "member-operator", "kubesaw-member-operator")
+	})
```

Comment:
the webhook shouldn't be deployed in the second member namespace - we should have only one webhook running in the whole cluster.